packages/storybook-builder-vite/codegen-iframe-script.js

```diff
@@ -13,11 +13,16 @@ function replaceCJStoESMPath(entryPath) {
 module.exports.generateIframeScriptCode = async function generateIframeScriptCode(
     options
 ) {
-    const { presets, configDir, framework } = options;
+    const { presets, configDir, framework, frameworkPath } = options;
```

Comment:
Does frameworkPath come as a config option from Storybook core? What's the use case for being able to override it?

Reply:
The use case is discussed in https://github.com/storybookjs/storybook/issues/12075, and support in webpack was added in https://github.com/storybookjs/storybook/pull/12087.  I _think_ these are the same options that the webpack builder also gets from storybook, right?

Reply:
Ah, I see. Thanks!